<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServeurWidget</class>
 <widget class="QWidget" name="ServeurWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServeurWidget</string>
  </property>
  <widget class="QLabel" name="label_numPort">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>76</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Num√©ro de port</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_numPort">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>72</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="value">
    <number>1025</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_LancementServeur">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>106</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Lancement serveur</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Clients">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>139</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Clients</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Clients">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>163</y>
     <width>291</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Quitter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>ServeurWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
